### DISCUSSION TITLE
Welcome to NiChart Software Testing!

### DISCUSSION THREAD

**Discussion by gurayerus** [CBICA/NiChart_Project#66] [Testing Announcements]
Hi everyone,

Thanks for volunteering to test NiChart!

This section is for keeping you updated on the testing process and for sharing discussions and feedback. Feel free to share feedback and participate in discussions.

For bug reports, please open an issue on the Issues tab.

We appreciate your participation!

**Comment by A-Cronnex** [comment]
Hello, my name is Aaron. I am future researcher from the Univeristy of Panama. Two of the members of your project have personally recommended the usage of your pipeline, I have talked with two of them in the NITRC forum. However, I cannot use the tool in the same way as the tutorial shows because I can't choose the output directory based on my own data and neither I can choose the type of training I wanna do (In the DLMUSE part, I can't chose between cuda and the others) I am very eager to see if your tool can help me with my research project, but I don't know why it's missing such key options that appear in the tutorial.
